core/nodejsActionBase/runner.js

```diff
@@ -46,6 +46,17 @@ function initializeActionHandler(message) {
                     return Promise.reject('Zipped actions must contain either package.json or index.js at the root.');
                 }
 
+                // install npm modules during init if source code zip doesn´t containt them
```

Comment:
This is nice!

Reply:
From my tests so far the current functionality is not impacted

Reply:
No edge case comes up in my mind too.